spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -145,7 +145,8 @@
               "identifier": "",
               "scheme": ""
             }
-          }]
+          }],
+          "communities": []
```

Comment:
Moving this to the metadata.

Reply:
It's good where it is. Naming should be changed though.